crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -202,13 +202,37 @@ type EndOfEpochTransaction {
 }
 
 type Epoch {
+	"""
+	The epoch's id
+	"""
 	epochId: Int!
+	"""
+	The reference gas price for this epoch
```

Comment:
Can you copy the explanation I added for the reference gas price in the system state summary here?

Reply:
Totally, thanks for suggesting it.